packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -42,6 +42,9 @@ const PatientBanner: React.FC<PatientBannerProps> = ({ patient, patientUuid, hid
   }, []);
 
   const isDeceased = Boolean(patient?.deceasedDateTime);
+  // render details button below patient details for workspaces
+  // 520px is the maximum width a workspace occupies
+  const renderDetailsButtonBelow = patientBannerRef.current?.scrollWidth <= 520;
```

Comment:
A naming suggestion:

```suggestion
	const maxWorkspaceWidth = 520;
	
  const showDetailsButtonBelowHeader = patientBannerRef.current?.scrollWidth <= maxWorkspaceWidth;
```

Reply:
The current logic doesn't account for two of the conditions Ian suggested.